activities/views/posts.py

```diff
@@ -53,6 +54,22 @@ def get_context_data(self):
             "link_original": True,
             "ancestors": ancestors,
             "descendants": descendants,
+            "opengraph": {
+                "title": f"{self.post_obj.author.name} (@{self.post_obj.author.handle})",
+                "type": "article",
+                "published_time": self.post_obj.published.isoformat()
+                or self.post_obj.created.isoformat(),
+                "modified_time": self.post_obj.edited.isoformat()
+                or self.post_obj.published.isoformat()
+                or self.post_obj.created.isoformat(),
```

Comment:
If you wrap these `or` sets in parentheses, the auto-formatter will indent them, which I think is more readable.

Reply:
Ah, ok. Will do!